.evergreen/scripts/generate-parallel-e2e-tasks.js

```diff
@@ -139,16 +140,30 @@ const generateParallelE2ETasks = (bv) => {
 };
 
 const main = () => {
-  const buildVariant = process.env.BUILD_VARIANT;
+    const buildVariant = process.env.BUILD_VARIANT;
+
+  // Check if there are any changes in the spruce directory
```

Comment:
the comment is a bit confusing so I would suggest expanding on it or removing it